sections/faqs/browser-support.js

```diff
@@ -0,0 +1,11 @@
+import md from "components/md"
+
+const BrowserSupport = () => md`
+  ## Which browsers does styled-components support?
+
+  Styled-components supports all the browsers that ReactDOM supports.
```

Comment:
- Remove "styled-components" (see above, btw styled-components is never capitalised) and use passive tone, i.e "All browsers that ~ReactDOM~ React supports are supported."
- Use "React" as that's what their page does as well: https://reactjs.org/docs/react-dom.html#browser-support
 
Also, please add a paragraph after this sentence. More paragraphs help readability most of the time.